routes/main.ts

```diff
@@ -56,19 +47,13 @@ const history: TabScreen = {
 export const mainRoutes: TabScreen[] = [];
 mainRoutes.push(home);
 mainRoutes.push(scan);
-
```

Comment:
The request screen should be shown only for android and not for iOS. can you please add the necessary checks, and if we are not showing the "receive cards" option in settings screen for ios, how are we going to show only "received cards" option. have we got the confirmation from pragya reg this ?

Reply:
Resolved by adding conditional rendering in settings.tsx

Reply:
Resolved by doing conditional rendering in SettingScreen.tsx